### ISSUE TITLE
[Feature Request] Increase Leaderboard to show Top 10 Contributors

### ISSUE THREAD

**Issue by dfuentes87** [Wake-Tech-Programming-Club/wtpc-bot#13]
**Is your feature request related to a problem? Please describe.**

With the increase of active users in the Programming Club, I feel the Leaderboard should show the Top 10 Contributors

**Describe the solution you'd like**

Increase Leaderboard from Top 5 to Top 10

**Describe alternatives you've considered**

n/a

**Additional context**

n/a

**Comment by victoriacheng15** [comment]
Can I work on this?

**Comment by dfuentes87** [comment]
@victoriacheng15 Definitely, go for it! However be aware of Issue #12 which might mess with any testing you may do

**Comment by victoriacheng15** [comment]
> @victoriacheng15 Definitely, go for it! However be aware of Issue #12 which might mess with any testing you may do

@dfuentes87 

I followed the installation steps, I got an error `401: unauthorized`. So not sure how I can test if this work or not.

The only thing I changed is the number 5 to 10 in the code below. 
`const users = await db.query('SELECT username, points FROM users ORDER BY points DESC LIMIT 10');`

So let me know if it is ok to push the change without testing.

**Comment by dfuentes87** [comment]
@aahmad4 Can you help troubleshoot the installation issue above please

**Comment by aahmad4** [comment]
Hi @victoriacheng15, thanks for offering to tackle this feature request. Could you please paste the error you're getting so we can try and figure out how to help here?

**Comment by aahmad4** [comment]
@seanthedev
